classes/SuperDirt.sc

```diff
@@ -284,14 +284,22 @@ SuperDirt {
 			}
 		};
 
-
 		netResponders.add(
+			OSCFunc({ |msg, time, tidalAddr|
+				var name = msg[1].asSymbol;
+				var synth = SynthDescLib.global.at(name);
+				var controls = synth.controls;
+				var controlNames = msg[1];
+				controls.do{arg control; controlNames = controlNames ++ " " ++ control.name};
+				tidalAddr.sendMsg("/dirt/synth-info/reply", controlNames);
+				controlNames.postln;
```

Comment:
there is still a postln left in, probably we don't want this?

Reply:
oops, removed that now